UofT-Frosh-Orientation/orientation-website
# #707 - Scunt Team Services Tests

## **Contribution:**

### **Issue:**
Closes #707 

### **Accomplishments:**
- Made tests for functions in `ScuntTeamServices.js`: `updateLeaderTeam`, `getTeamPoints`, `getTeams`, `bribeTransaction`, `getScuntJudges`, `refillBribePoints`, `addTransaction`, `subtractTransaction`, `viewTransactions`, `checkTransaction`, `initializeTeams`, `deleteTransaction`, `viewRecentTransactions`, and `setTeamName`.

### **Visuals:** 
n/a

## **Details:**

### **Expected Behavior:**
- All tests should pass for the functions not commented out

### **Testing Steps:**
- Run docker command, cd server, yarn test
- Check if all tests pass for `ScuntTeamServices.test.js`

### **Complications:**
- Some tests don't work for `initializeTeams`, `bribeTransaction`, and `deleteTransaction` (they are commented out)
- Not sure how to test other logic for addTransaction where the assert checks for 'Updated to...' in the transaction

### **Resources:**
n/a
